Source/Core/Core/ConfigManager.h

```diff
@@ -4,10 +4,12 @@
 
 #pragma once
 
+#include <algorithm>
```

Comment:
Is this still needed?

Reply:
Pretty sure I use it in line 854 when I do `    std::replace(executable_path.begin(), executable_path.end(), BACKSLASH, FORWARDSLASH);
`

Reply:
Although I also have the algorithm include in the `ConfigManager.h` file, so maybe not.

Reply:
Then the answer is no, this should be removed from the header since std::replace is only used in the .cpp.